### ISSUE TITLE
Onion Launcher not working from a fresh install

### ISSUE THREAD

**Issue by btrsco** [jimgraygit/Onion#109]
I recently got my Miyoo Mini, I did a fresh install from stock to OnionOS v3.54. I have been playing it without many issues and have been wanting to use the Onion Launcher.

I have enabled and disabled this app multiple times without being sent to another screen, just changes its name. I asked on Reddit and someone suggested that I just enable the launcher, start a game and then turn off the device and restart it to see the "magic". When I do that, I still don't see any differences - it just restarts as normal. Am I doing something wrong or is the launcher itself broken/bugged?

I also tried to find videos and articles about this feature but it doesn't seem to be documented anywhere, even the wiki.

Thanks for any help on this!

**Comment by IllustratedMan-code** [comment]
For me, when the launcher is enabled, if I enter a game, then exit the game, then turn the device off, once I turn it on again it boots me right into the game. The launcher itself will not send you to a different screen when you enable it.

**Comment by btrsco** [comment]
> For me, when the launcher is enabled, if I enter a game, then exit the game, then turn the device off, once I turn it on again it boots me right into the game. The launcher itself will not send you to a different screen when you enable it.

Just tried this and still no luck… these are the exact steps I did, let me know if I’m doing something wrong:

* Apps > Open “Onion Launcher - Disabled” (name changes to “Enabled” and icon to green)
* Recents > Open “Pokémon Fire Red”
* Close Pokémon (Menu + Start)
* Hold Power Button > Press A to Turn off
* Hold Power Button to turn back on
* See Miyoo Logo then I see the Home Screen of OnionOS where I can go to Recents, Faves, Games etc. 

Nothing different happens for me. I’ve tried with multiple games from multiple systems. The Launcher still says it’s enabled between restarts, can’t seem to get this to work.

**Comment by btrsco** [comment]
I also had my girlfriend go through the same steps on her Miyoo Mini and it didn’t work either. Not sure what to do, I’m gonna try to wipe and reinstall Onion v3 on mine again and see if that works.

**Comment by BenKesselring** [comment]
You don’t happen to have deleted the `.tmp_update` folder, have you? I’ve seen a few people now who have deleted that folder, and they had similar issues with getting Onion Launcher working.

**Comment by Olywa** [comment]
I think maybe you are adding an unnecessary step (_Close Pokémon (Menu + Start)_). 
Once Onion Launcher is enabled you do not need to exit the game before power off. You just power off and Onion remembers the last game you were in on boot.

**Comment by IllustratedMan-code** [comment]
I can't reproduce the issue with those steps. Maybe the app directory is messed up. Check to see if App/OnionLauncher/data/.enabled exists when the app is enabled (you can use the file browser app to check)

**Comment by btrsco** [comment]
Was writing a long response to everyones point, but this got me thinking:
> You don’t happen to have deleted the `.tmp_update` folder, have you?

Initially I thought you were talking about the `.tmp_update` directory from the `App/OnionLauncher/data/`, but I tried that without any change. Then I decided to read the instructions from the README for the install and did a full reinstall just to be safe and didn't delete the `.tmp_update` this time. I watched a video mentioning that you were supposed to delete that, but now that I reinstalled and didn't delete it its working fine. Crazy lol, I'm sorry for the wasted time! Thanks for the input though, otherwise I wouldn't have thought to re-read the instructions.

**Comment by btrsco** [comment]
Was writing a long response to everyones point, but this got me thinking:
> You don’t happen to have deleted the `.tmp_update` folder, have you?

Initially I thought you were talking about the `.tmp_update` directory from the `App/OnionLauncher/data/`, but I tried that without any change. Then I decided to read the instructions from the README for the install and did a full reinstall just to be safe and didn't delete the `.tmp_update` this time. I watched a video mentioning that you were supposed to delete that, but now that I reinstalled and didn't delete it its working fine. Crazy lol, I'm sorry for the wasted time! Thanks for the input though, otherwise I wouldn't have thought to re-read the instructions.
